### PR TITLE
Utils

### PR DESCRIPTION
**Pull request by j2kan**
1. updated server with this PR and the security stuff.
Approach:
this util file helps load my environment variables from my configuration file which has been stored in /var/www 
2. changed secret key
3. cleaned up all the passwords and sensitive information to the config file.

### CODE REVIEW

**Review by smwatts — COMMENTED** [review]
Im good with the changes for files 1&2, mind adding a description of 3 in the PR? So I can follow :)

File: taworks/taworks/utils.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            old_umask = os.umask(0o177)  # Use '0600' file permissions
+            config = ConfigParser()
+            config.add_section('django')
+            config['django']['secret_key'] = get_random_string(key_length, VALID_KEY_CHARS)
+
+            with open(envfile, 'w') as configfile:
+                config.write(configfile)
+            os.umask(old_umask)
+
+        if (os.stat(envfile).st_mode & 0o777) != 0o600:
```
Comment (j2kan) [inline taworks/taworks/utils.py:32]: checks to make sure file is correct permissioning

**Review by smwatts — APPROVED** [review]
LGTM!

**Review by amyleblond — APPROVED** [review]
agreed with @smwatts , lets go over the utils.py file wednesday for all the learnings

**Pull request merged**